packages/jest-transform/src/ScriptTransformer.ts

```diff
@@ -89,6 +87,7 @@ class ScriptTransformer {
     {transformer: Transformer; transformerConfig: unknown}
   >();
   private _transformsAreLoaded = false;
+  private _babelPluginIstanbul: unknown = null;
```

Comment:
- Can we use `undefined` here instead of `null`?
- Do we have any existing typescript type which can represent the interface of a Babel plugin?